data/light.json

```diff
@@ -0,0 +1,118 @@
+{
+  "colors": {
+    "text": {
+      "base": {
+        "value": "{colors.grey.800}",
+        "type": "color"
+      },
+      "emphasis": {
+        "value": "{colors.grey.700}",
+        "type": "color"
+      },
+      "subdued": {
+        "value": "{colors.grey.500}",
+        "type": "color"
+      }
+    },
+    "surface": {
+      "surfaceLevel0": {
```

Comment:
Are we naming this because we are hoping to strip out the `surface` with a transform?